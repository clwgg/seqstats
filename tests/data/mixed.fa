>empty

>short
A
>medium
ACGT
>long
ACGTACGTACGT
