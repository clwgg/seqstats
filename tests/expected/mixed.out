Total n:	3
Total seq:	17 bp
Avg. seq:	5.67 bp
Median seq:	4.00 bp
N 50:		12 bp
Min seq:	1 bp
Max seq:	12 bp
